src/parser/creative_linear_parser.js

```diff
@@ -158,6 +159,27 @@ export function parseCreativeLinear(creativeElement, creativeAttributes) {
 
           creative.mediaFiles.push(mediaFile);
         });
+
+      const mezzanineElement = parserUtils.childByName(mediaFilesElement, 'Mezzanine');
+      if (mezzanineElement &&
+        mezzanineElement.getAttribute('delivery') &&
+        mezzanineElement.getAttribute('type') &&
+        mezzanineElement.getAttribute('width') &&
+        mezzanineElement.getAttribute('height')) {
```

Comment:
I think you should move these conditions to a separate function called i.e. `getMezzanineRequiredAttributes` and inside that function return either the object with the required attributes or null. 
It will make this part of the code more explicit as to what is the condition for.